### ISSUE TITLE
Update neighbourhood records in Singapore

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1092]
### tl;dr

Status:
- [ ]  Propose sources
- [ ]  Establish concordances between WOF and source
- [ ]  Upgrade / add WOF shapes
- [ ]  Add new or missing features to WOF
- [ ]  Deprecate / supersede WOF features
- [ ]  Draft review
- [ ]  Draft revisions based on review comments
- [ ]  Final review
- [ ]  Final revisions based on review comments
- [ ]  Import into Who's On First
### Details

Sourcing:
- Permissive license?
- Add new src to Who's On First

Upgrade / add:
- `macrohood`
- `neighbourhood`
- `microhood`

Add new or missing features to WOF
- `macrohood`
- `neighbourhood`
- `microhood`

Deprecate / supersede:
- `macrohood`
- `neighbourhood`
- `microhood`

**Comment by stepps00** [comment]
Singapore has "planning divisions" which are, for the most part, used as "neighbourhoods" or district areas in Singapore.

WOF currently has just one neighbourhood record, which will likely be deprecated: https://spelunker.whosonfirst.org/id/85784203/

**Planning division information**: https://en.wikipedia.org/wiki/Planning_Areas_of_Singapore
**Planning division data**: https://data.gov.sg/dataset/planning-area-census2010
**License**: https://data.gov.sg/open-data-licence

**Comment by missinglink** [comment]
I had a look at this today, these PA (Planning Areas) are not exactly what the locals would refer to as neighbourhoods.

For instance many online travel websites will talk about 'Sentosa' as a neighbourhood but the PA for that is 'Southern Islands'.
Another example on the mainland (main island!?) is 'Chinatown' in the PA of 'Outram'.

Just wanted to highlight that, I think someone will need to make a call as to whether these are modelled as `macrohood/neighbourhood` or as `neighbourhood/microhood`.

These 'subzones' are listed at the bottom of the PA Wiki Page (subzones in parenthesis):
https://en.wikipedia.org/wiki/Planning_Areas_of_Singapore

<img width="1226" alt="Screenshot 2021-02-10 at 12 18 53" src="https://user-images.githubusercontent.com/738069/107441584-57f8f080-6b9a-11eb-8543-4c2fbae349a3.png">

**Comment by missinglink** [comment]
Sources: https://data.gov.sg/search?q=boundary+2019

#### Master Plan 2019 Planning Area Boundary (No Sea)

<img width="849" alt="Screenshot 2021-02-10 at 13 16 15" src="https://user-images.githubusercontent.com/738069/107445961-5c290c00-6ba2-11eb-92c5-d3bd6c88d80e.png">

### Master Plan 2019 Subzone Boundary (No Sea)

<img width="800" alt="Screenshot 2021-02-10 at 13 15 20" src="https://user-images.githubusercontent.com/738069/107446005-72cf6300-6ba2-11eb-9927-6fc7633e4af6.png">

**Comment by nvkelso** [comment]
@stepps00 can you take a fresh look, please?

**Comment by stepps00** [comment]
This was complete through https://github.com/whosonfirst-data/whosonfirst-data-admin-sg/pull/9, new source here: https://github.com/whosonfirst/whosonfirst-sources/blob/master/sources/sg-sggov.json

Closing.
